.github/workflows/layered-build.yaml

```diff
@@ -16,4 +16,16 @@ jobs:
                   path: element-web/webapp
                   # We'll only use this in a triggered job, then we're done with it
                   retention-days: 1
+            - uses: actions/github-script@v3.1.0
+              with:
+                script: |
+                    var fs = require('fs');
+                    fs.writeFileSync('${{github.workspace}}/context.json', JSON.stringify(context));
```

Comment:
it's a little uncomfortable to be putting the whole context here (just in case tokens get sent through). Could we limit it to just the fields we need as a sort of custom carrier object?

Reply:
good point - I've just stripped it to the PR object which is a standard API shape of info about the pull request itself